packages/inbound-mail-parser/src/parser.js

```diff
@@ -31,8 +31,8 @@ function createAttachment(file) {
  * @param {Object} config inbound configuration object
```

Comment:
Change @param names in the comments wrt to the params passed in the function

Reply:
The parameter which is passed is actually file object only. It is just destructured, so I don't think there is any need to update the param.

Reply:
As per esdoc [https://esdoc.org/manual/tags.html#object](https://esdoc.org/manual/tags.html#object) It should be written in this manner.

Reply:
Oh I see, but as per that doc, even in the original case the object keys have to be mentioned, but that were skipped. So I think it would be much better if a new issue is opened which is meant to improve the inline code documentation. Because this applies to all packages.

Reply:
esdoc has been added to the repo as of #549 and an issue has been added to clean up the docs. 

Thank you both!!